src/WebJobs.Extensions.DocumentDB/Trigger/CosmosDBTriggerAttributeBindingProvider.cs

```diff
@@ -144,5 +160,55 @@ private bool TryReadFromSettings(string settingsKey, string defaultValue, out st
             settingsValue = _nameResolver.Resolve(settingsKey);
             return !string.IsNullOrEmpty(settingsValue);
         }
+
+        private static async Task CreateIfNotExistAsync(IDocumentDBService service, string databaseName, string collectionName, int throughput)
+        {
+            await DocumentDBUtility.RetryAsync(() => CreateDatabaseIfNotExistsAsync(service, databaseName),
+                DefaultMaxThrottleRetries, codesToIgnore: HttpStatusCode.Conflict);
```

Comment:
FYI -- I opened https://github.com/Azure/azure-webjobs-sdk-extensions/issues/280 to track the removal of `RetryAsync` from the rest of the project.